src/renderer/containers/map/infoBox/index.jsx

```diff
@@ -38,11 +36,11 @@ export default class InfoBox extends React.Component {
       subject: 'Service Location No Longer Active',
       body: `The following service area is no longer active.\n\n\t${
         details.name
-      }\n\t${details.address1}\n\t${details.address2}\n\tPhone: ${
-        details.areaCode
-      }${
-        details.phoneNumber
-      }\n\nDataset: https://discovery.smartcolumbusos.com/dataset/handson_central_ohio/570a8e02_fb0e_4cee_895b_3b32bd740650`,
+      }\n\t${details.address1}\n\t${
+        details.address2
+      }\n\tPhone(s): ${details.phones.join(
+        ', '
+      )}\n\nDataset: https://discovery.smartcolumbusos.com/dataset/handson_central_ohio/community_services_agencies`,
```

Comment:
I believe there is a const for this dataset url somewhere

Reply:
Only in the API :(